tools/azure-pipelines/build-python-wheels.yml

```diff
@@ -0,0 +1,68 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+jobs:
+  - job: compile_${{parameters.stage_name}}
+    pool:
+      vmImage: 'ubuntu-16.04'
+    container: flink-build-container
+    workspace:
+      clean: all
+    steps:
+      # Compile
+      - script: STAGE=compile ${{parameters.environment}} ./tools/azure_controller.sh compile
+        displayName: Build
+
+      # upload artifacts for building wheels
+      - task: PublishPipelineArtifact@1
+        inputs:
+          path: $(CACHE_FLINK_DIR)
+          artifact: FlinkCompileCacheDir
```

Comment:
```suggestion
          artifact: FlinkCompileCacheDir-${{parameters.stage_name}}
```

Reply:
See comment below

Reply:
I will correct it.

Reply:
I have addressed it at the latest commit